hw/dv/tools/dvsim/verilator.hjson

```diff
@@ -70,7 +70,7 @@
                ]
 
   run_opts: [// Set random seed.
-             // "+verilator+seed+{seed}",
+             // "+verilator+seed+{svseed}",
```

Comment:
Do we need this? Can we just remove these comments altogether?

Reply:
Sure I'll remove it.